recipes/compositionspace/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "compositionspace" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/compositionspace-{{ version }}.tar.gz
+  sha256: addb8a145c921ebbbfd92723cfed1df2d5ccde4279fbc44220ee276a32e252b1
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```

Comment:
Are you skipping Windows b/c of a missing dependency?  This looks it can be noarch and a failure b/c of a  missing is fine.

Reply:
Yes, that was indeed the case. Now I put it back to noarch

Reply:
BTW, looks like pyvista is available on Windows. The problem is with its metadata and `pip check` is failing. We need to investigate that. Regardless, this PR is ready to do and there is nothing that you need to do here. If/when the pyvista issue is solved this package will be installable on Windows without the need for any changes here.